File: JobFinder-CLI/src/main/java/com/ofdun/jobfinder/cli/dto/profiles/Employer.java
package com.ofdun.jobfinder.cli.dto.profiles;

import com.ofdun.jobfinder.cli.dto.common.Location;

public record Employer(
        Long id,
        String name,
        String email,
        String description,
        String address,
        String websiteLink,
        Location location) {}

File: JobFinder-front/src/shared/types/employer.ts
export interface Employer {
  id: number;
  name: string;
  email: string;
  description: string;
  address: string;
  websiteLink: string;
  location?: {
    id: number;
    city: string;
    country: string;
  };
}

export interface EmployerUpdateRequest {
  name: string;
  email: string;
  password?: string;
  description: string;
  address: string;
  websiteLink: string;
  locationId: number;
}

